packages/ui/src/ui/store/reducers/navigation/url-mapping.ts

```
@@ -74,7 +75,16 @@ export const navigationParams: LocationParameters = {
             shouldPush: true,
         },
     },
-};
+    ...UIFactory.getNavigationExtraTabs().reduce((acc, extraTab) => {
+        if (extraTab.getParams) {
+            acc = {
+                ...acc,
+                ...extraTab.getParams(),
```

It is better to throw an error in case of duplicated keys.